src/CoreHandler.php

```diff
@@ -17,10 +17,13 @@ class CoreHandler extends Object implements SelfHandlingCommand
 
     /**
      * Load Stripe private key from .env
+     *
+     * @param \Stripe\Stripe $stripe
+     * @param string $stripe_private_key
      */
-    public function init()
+    public function init(Stripe $stripe, string $stripe_private_key)
     {
-        \Stripe\Stripe::setApiKey(getenv('STRIPE_PRV_KEY'));
+        $stripe::setApiKey($stripe_private_key);
```

Comment:
So this is a little strange - while I can see that this works, this would also work if `$stripe` were set to `'Stripe'`. Could we use the typical object operator here, `$stripe->setApiKey()`?

Reply:
Stripe library req. static call.

Reply:
That really is a bummer. So, fun fact, PHP lets you access static methods on instances, but if `setApiKey()` is a static method, I'd rather be used as such.

Reply:
Actually looked this up. https://github.com/stripe/stripe-php/blob/d1978a7f20cdbd05c5c462ffa6728f279f03e0c9/lib/Stripe.php#L87 Does allow instantiating the \Stripe\Stripe class and calling `setApiuKey()` dynamically. 

However, the official documentation for the usage of the dependant class show static setting of the class properties https://github.com/stripe/stripe-php . I'd prefer to stick with the implementation pattern provided.